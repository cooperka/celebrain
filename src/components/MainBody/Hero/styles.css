.Hero {
  min-height: 30rem;
  margin: 4rem 0;
  display: flex;
  flex-direction: column;
  font-weight: 500;
}

.Hero .title {
  margin: 1rem;
  font-size: 5rem;
}

.Hero .subtitle {
  margin: 1rem auto;
  font-size: 2.2rem;
  max-width: 44rem;
}

.Hero .buttons {
  margin: 1rem auto;
}
