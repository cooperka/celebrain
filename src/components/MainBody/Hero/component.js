// @flow

import React from 'react';

import NavButton from '../../NavButton/index';

import './styles.css';

function Hero() {
  return (
    <div className="Hero">
      <div className="title">Lovable. Simple. Refined.</div>
      <div className="subtitle">
        I listen to users and build the <i>minimal&nbsp;lovable&nbsp;solution</i> to meet their needs.
      </div>
      <div className="buttons">
        <NavButton bordered externalLinkTo="mailto:kevin@lsrfocus.com">Contact</NavButton>
      </div>
    </div>
  );
}

export default Hero;
