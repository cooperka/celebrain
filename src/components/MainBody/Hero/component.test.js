// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Hero from './component';

it('renders', () => {
  const tree = renderer.create(<Hero />);
  expect(tree.toJSON()).toMatchSnapshot();
});
