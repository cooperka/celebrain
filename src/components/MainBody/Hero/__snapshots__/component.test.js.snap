// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="Hero"
>
  <div
    className="title"
  >
    Lovable. Simple. Refined.
  </div>
  <div
    className="section"
  >
    <div
      className="subtitle"
    >
      I listen to users and build the 
      <i>
        minimal lovable solution
      </i>
       to meet their needs.
    </div>
  </div>
  <div
    className="buttons"
  >
    <a
      className="MuiButtonBase-root-16 MuiButton-root-1 NavButton bordered"
      exact={undefined}
      href="mailto:kevin@lsrfocus.com"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      to={undefined}
    >
      <span
        className="MuiButton-label-2"
      >
        <div
          className="link-text"
        >
          Contact
        </div>
      </span>
      <span
        className="MuiTouchRipple-root-18"
      />
    </a>
  </div>
</div>
`;
